templates/common-templates.adoc

```diff
@@ -167,3 +168,116 @@ Please note that, after the generation step, VM objects and template
 objects have no relationship with each other besides the aforementioned
 label (e.g. changes in templates do not automatically affect VMs, or
 vice versa).
+
+common template customization
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The templates provided by the kubevirt project provide a set of
+conventions and annotations that augment the basic feature of the
+https://docs.okd.io/latest/dev_guide/templates.html[openshift
+templates]. You can customize your kubevirt-provided templates editing
+these annotations, or you can add them to your existing templates to
+make them consumable by the kubevirt services.
+
+Here’s a description of the kubevirt annotations. Unless otherwise
+specified, the following keys are meant to be top-level entries of the
+template metadata, like
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Template
+metadata:
+  name: windows-10
+  annotations:
+    openshift.io/display-name: "Generic demo template"
+----
+
+All the following annotations are prefixed with
+`defaults.template.cnv.io`, which is omitted below for brevity. So the
+actual annotations you should use will look like
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Template
+metadata:
+  name: windows-10
+  annotations:
+    defaults.template.cnv.io/disk: default-disk
+    defaults.template.cnv.io/volume: default-volume
+    defaults.template.cnv.io/nic: default-nic
+    defaults.template.cnv.io/network: default-network
+----
+
+Unless otherwise specified, all annotations are meant to be safe
+defaults, both for performance and compability, and hints for the
+CNV-aware UI and tooling.
+
+disk
+++++
+
+Name of the default disk to use.
+
+Example:
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Template
+metadata:
+  name: Linux
+  annotations:
+    defaults.template.cnv.io/disk: virtio
```

Comment:
You left virtio there and the example might be nicer if you kept the disk section as well so the name reference is demonstrated.

Reply:
Done